src/gp/lti_sde.jl

```diff
@@ -143,44 +143,38 @@ end
 
 # Generic constructors for base kernels.
 
-function lgssm_components(
-    k::SimpleKernel, t::AbstractVector{<:Real}, storage::StorageType{T},
-) where {T<:Real}
-
-    # Compute stationary distribution and sde.
-    x0 = stationary_distribution(k, storage)
-    P = x0.P
-    F, _, H = to_sde(k, storage)
-
-    # Use stationary distribution + sde to compute finite-dimensional Gauss-Markov model.
+function broadcast_components((F, q, H)::Tuple, x0::Gaussian, t::AbstractVector{<:Real}, ::StorageType{T}) where {T}
+    P = Symmetric(x0.P)
     t = vcat([first(t) - 1], t)
     As = _map(Δt -> time_exp(F, T(Δt)), diff(t))
     as = Fill(Zeros{T}(size(first(As), 1)), length(As))
-    Qs = _map(A -> Symmetric(P) - A * Symmetric(P) * A', As)
+    Qs = _map(A -> P - A * P * A', As)
```

Comment:
Why did this change?

Reply:
4 lines above, I already define `P` to be `Symmetric` (instead of doing it for every operation

Reply:
🤦 so you did